docs/sources/developers/plugins/publish-a-plugin.md

```diff
@@ -1,148 +1,44 @@
 ---
 aliases:
   - share-a-plugin/
-  - package-a-plugin/
 description: Learn how to package and share your plugin.
 title: Publish a plugin
 type: docs
 ---
 
-# Publish a plugin
+# Publish or update a plugin
 
-You've just built your first plugin, and now you want to share it with the world.
-In this guide, you'll learn how to package and share your plugin with others.
+You've just built your first plugin and [packaged it]({{< relref "package-a-plugin.md" >}}), and now you want to share it with the world.
+In this guide, you'll learn how to share your plugin with others.
 
-For Grafana to be able to load a plugin, it first needs to be built.
-When you build a plugin from source, a `dist` directory is created that contains the production build, or _plugin assets_, for your plugin.
-
-When the Grafana server starts, it recursively looks in the plugin directory for any directory that contains a `plugin.json` file and tries to load the plugin assets in the same directory.
-
-There are three steps needed to package a plugin:
-
-- Building the plugin
-- Signing the plugin
-- Packaging the plugin
-
-1. Build the plugin
-
-   ```
-   yarn install --pure-lockfile
-   yarn build
-   ```
-
-1. (Optional) If your data source plugin has a backend plugin, build it as well.
-
-   ```
-   mage
-   ```
-
-   make sure that all the binaries are executable and have a `0755` (`-rwxr-xr-x`) permission
-
-1. [Sign the plugin]({{< relref "./sign-a-plugin" >}}).
-
-1. Rename `dist` directory to match your plugin id and create a ZIP archive
-
-   ```
-   mv dist/ myorg-simple-panel
-   zip myorg-simple-panel-1.0.0.zip myorg-simple-panel -r
-   ```
+The best way to share your plugin with the world is to publish it in the [Grafana plugin catalog]({{< "/plugins" >}}).
+By having your plugin published on Grafana.com, more users will be able to discover your plugin.
 
-1. (Optional) Verify your plugin is packaged correctly using [zipinfo](https://linux.die.net/man/1/zipinfo).
-   It should look like this:
+## Before you begin
 
-```
-zipinfo grafana-clickhouse-datasource-1.1.2.zip
+Before you begin to publish a plugin, it needs to be built and packaged. 
```

Comment:
"it needs to be built and packaged" and the first sentence in the section above feel a bit redundant?

Reply:
same with package as we say "you've just built your first plugin and packaged it" already at the top?

Reply:
Fix checked in.